### PR TITLE
More context on backend.xml in barceloneta so themers understand when to use it

### PR DESCRIPTION
**Pull request by djay**
Fixes #.

Improves:

Changes proposed in this pull request:
## 
## 
##

### CODE REVIEW

File: adapt-and-extend/theming/barceloneta.rst [hunk 1]
```diff
@@ -144,6 +144,29 @@ You can develop a custom Diazo based theme and use the Barceloneta theme only fo
 
 You can define your own Diazo bundle (JavaScript and Less/CSS) in your manifest.cfg file by using the options ``development-js``, ``production-js``, ``development-css`` and ``production-css``. This bundle will not be included in the backend theme.
 
+
+Current issues:
+
```
Comment (svx) [inline adapt-and-extend/theming/barceloneta.rst:149]: Would it be an idea to use a subheading here, like

```
Current issues
^^^^^^^^^^^^^^  
```

File: adapt-and-extend/theming/barceloneta.rst [hunk 2]
```diff
@@ -144,6 +144,29 @@ You can develop a custom Diazo based theme and use the Barceloneta theme only fo
...
+Why this is a good idea:
+
+- It reduces the effort in theming. In most cases your users will never see edit, sharing, sitesetup or other aspects of the Plone backend UI. Making those screens work with a new theme is a lot of work.
+- The backend pages can include a lot of add on functionality which might be hard to integrate. This might not be tested for integration into third-party themes.
+- Barceloneta has been tested for UI and to some extend accessibility. Retheming could make the UI harder to use for editor.
+- The backend UI is more likely to change between versions. Theming it means your theme will have to change too.
+
+How this works:
+
```
Comment (svx) [inline adapt-and-extend/theming/barceloneta.rst:160]: and here

```
How this works
^^^^^^^^^^^^^^
```

**Comment by svx** [comment]
Would it an idea to use subheadings ?

```
Heading 3
^^^^^^^^^^
```

would fit here

More in our style-guide: http://docs.plone.org/about/rst-styleguide.html#id8

**Comment by svx** [comment]
What is the status here ?

**Comment by jensens** [comment]
@djay I would like to see this in the docs!

**Comment by svx** [comment]
@djay what is the status here ? :)

**Comment by svx** [comment]
@djay ping !

**Comment by svx** [comment]
Ok so, lets say if someone would fix the rst syntax issues .. Would it be OK afterwards to merge into 5.1 ?

**Comment by jensens** [comment]
agree to merge after RST syntax fixes

**Comment by gforcada** [comment]
@svx @djay I fixed the syntax on https://github.com/plone/documentation/pull/746

**Comment by svx** [comment]
@gforcada I improved #746

**Comment by svx** [comment]
Just a moment, doing more improvements ...

**Comment by svx** [comment]
ok, added some more links for better info

**Comment by gforcada** [comment]
So let's close this one in favor of #746

**Pull request closed without merging**